RiotSharp.Test/RiotApiTest.cs

```diff
@@ -744,5 +745,35 @@ public void GetTotalChampionMasteryScoreAsync_Test()
             });
         }
         #endregion
+
+        #region Thrid Party Tests
+
+        [TestMethod]
+        [TestCategory("RiotApi")]
+        public void GetThirdPartyCode_Test()
+        {
+            EnsureCredibility(() =>
+            {
+                var code = Api.ThirdParty.GetThirdPartyCodeBySummonerId(Summoner1And2Region,
+                    Summoner1Id);
+
+                Assert.AreEqual(RiotApiTestBase.ThirdPartyCode, code);
+            });
+        }
+
+        [TestMethod]
+        [TestCategory("RiotApi"), TestCategory("Async")]
+        public void GetThirdPartyCodeAsync_Test()
+        {
+            EnsureCredibility(() =>
+            {
+                var code = Api.ThirdParty.GetThirdPartyCodeBySummonerIdAsync(Summoner1And2Region,
+                    Summoner1Id);
+
+                Assert.AreEqual(RiotApiTestBase.ThirdPartyCode, code.Result);
+            });
+        }
+
+        #endregion
```

Comment:
Do we want to keep those integration tests or just the mocks?

Reply:
Regarding this question : in my opinion we should keep both. Since integration test will cover our pieces and shows if it interacts well with this last; while moq tests intend to check if the logic implemented/ refactored works as expected or not.